collections/nemo_asr/nemo_asr/parts/manifest.py

```diff
@@ -157,35 +161,10 @@ def normalize_text(self, text, labels):
 
         # Turn all other punctuation to whitespace
         table = str.maketrans(punctuation, " " * len(punctuation))
-        text = self.english_string_normalization(
-            text,
-            labels=labels,
-            table=table,
-            punctuation_to_replace=punctuation_to_replace,
-            clean_fn=self.clean_text)
 
-        return text
-
-    @staticmethod
-    def english_string_normalization(s, labels, table, punctuation_to_replace,
-                                     clean_fn):
-        """
-        Normalizes string. For example:
-        'call me at 8:00 pm!' -> 'call me at eight zero pm'
-
-        Args:
-            s: string to normalize
-            labels: labels used during model training.
-            table: translation table of characters to replace
-            punctuation_to_replace: punctuation to be ignored in `table`
-            clean_fn: function to do the cleaning
-
-        Returns:
-            Normalized string
-        """
         try:
-            text = clean_fn(s, table, punctuation_to_replace)
-            return text
+            text = clean_text(text, table, punctuation_to_replace)
         except BaseException:
             print("WARNING: Normalizing {} failed".format(s))
-            return None
+
+        return text
```

Comment:
What happens if we catch BaseException? Would we crash because text is undefined?

Reply:
`text` in that case would just be the original input (first argument).

Reply:
The previous default here: https://github.com/NVIDIA/NeMo/blob/547a326369802f178e0be4da10ffea9ac0f30feb/collections/nemo_asr/nemo_asr/parts/manifest.py#L44
was to ignore that transcript if we couldn't normalize it, like NaNs in common voice. This is why we returned None instead of the unormalized string. Is there a reason to change the default behaviour or am I misunderstanding how the code works now?

Reply:
Removed by accident, fixing now